Apps/Sandcastle/gallery/ParticleSystem.html

```diff
@@ -0,0 +1,445 @@
+Cesium.knockout.applyBindings(viewModel, toolbar);
+
+var emitterModelMatrix = new Cesium.Matrix4();
+
+//Generate a random circular pattern with varying heights.
+function computeCirclularFlight(lon, lat, radius) {
+    var property = new Cesium.SampledPositionProperty();
+    for (var i = 0; i <= 360; i += 45) {
+        var radians = Cesium.Math.toRadians(i);
+        var time = Cesium.JulianDate.addSeconds(start, i, new Cesium.JulianDate());
+        var position = Cesium.Cartesian3.fromDegrees(lon + (radius * 1.5 * Math.cos(radians)), lat + (radius * Math.sin(radians)), Cesium.Math.nextRandomNumber() * 500 + 1750);
+        property.addSample(time, position);
+    }
+    return property;
+}
+
+//Compute the entity position property.
+var circularPosition = computeCirclularFlight(-112.110693, 36.0994841, 0.03);
+var staticPosition = Cesium.Cartesian3.fromDegrees(-112.110693, 36.0994841, 1000);
+
+var entity = viewer.entities.add({
+
+    //Set the entity availability to the same interval as the simulation time.
+    availability : new Cesium.TimeIntervalCollection([new Cesium.TimeInterval({
+        start : start,
+        stop : stop
+    })]),
+
+    //Load the Cesium plane model to represent the entity
+    model : {
+        uri : '../../SampleData/models/CesiumAir/Cesium_Air.gltf',
+        minimumPixelSize : 64
+    },
+
+    particleSystem: {
+        image: '../../SampleData/fire.png',
+
+        startColor: Cesium.Color.RED.withAlpha(0.7),
+        endColor: Cesium.Color.YELLOW.withAlpha(0.3),
+
+        startScale: viewModel.startScale,
+        endScale: viewModel.endScale,
+
+        minLife: viewModel.minLife,
+        maxLife: viewModel.maxLife,
+
+        minSpeed: viewModel.minSpeed,
+        maxSpeed: viewModel.maxSpeed,
+
+        minWidth: viewModel.particleSize,
+        minHeight: viewModel.particleSize,
+
+        maxWidth: viewModel.particleSize,
+        maxHeight: viewModel.particleSize,
+
+        // Particles per second.
+        rate: viewModel.rate,
+
+        bursts: [
+            {time: 5.0, min: 300, max: 500},
+            {time: 10.0, min: 50, max: 100},
+            {time: 15.0, min: 200, max: 300}
+        ],
+
+        lifeTime: 16.0,
+
+        emitter: new Cesium.CircleEmitter({
+            radius: 0.5
+        }),
+
+        emitterModelMatrix: Cesium.Matrix4.fromTranslationRotationScale(
+            new Cesium.TranslationRotationScale(
+                new Cesium.Cartesian3(viewModel.transX, viewModel.transY, viewModel.transZ),
+                Cesium.Quaternion.fromHeadingPitchRoll(Cesium.HeadingPitchRoll.fromDegrees(viewModel.heading, viewModel.pitch, viewModel.roll))
+            ),
+            emitterModelMatrix
+        ),
+
+        forces: [applyGravity]
+    },
+
+    position : staticPosition
+});
+viewer.trackedEntity = entity;
+
+
+var scene = viewer.scene;
+scene.debugShowFramesPerSecond = true;
+
+var gravityScratch = new Cesium.Cartesian3();
+
+function applyGravity(p, dt) {
```

Comment:
I wouldn't make it part of Cesium. It is here and as a code example for the force callbacks.